nix/sources.json

```diff
@@ -1,4 +1,16 @@
 {
+    "gomod2nix": {
+        "branch": "fix-regex",
+        "description": "Convert applications using Go modules to Nix expressions",
+        "homepage": null,
+        "owner": "yihuang",
+        "repo": "gomod2nix",
+        "rev": "af2ff6eb58f1ebbc374bc2fa0c23fa105b2d490f",
+        "sha256": "1268jddzg2kf8jbbkc4wsvpq39lldi6cfp6vsl0sclg1k2r10hc9",
+        "type": "tarball",
+        "url": "https://github.com/yihuang/gomod2nix/archive/af2ff6eb58f1ebbc374bc2fa0c23fa105b2d490f.tar.gz",
+        "url_template": "https://github.com/<owner>/<repo>/archive/<rev>.tar.gz"
```

Comment:
what's the difference from upstream? some fix in regex for macOS?

Reply:
yeah, this https://github.com/tweag/gomod2nix/pull/9